src/main/java/com/softexpert/raffle/DrawWinner.java

```diff
@@ -35,4 +35,14 @@ private String getRandomWinner(List<String> participants) {
 		return participants.get(position);
 	}
 
+	private List<String> buildFinalWinnersList(List<String> participants, int numberOfWinners) {
+		List<String> winners = new ArrayList<String>();
```

Comment:
Esse código ficou meio estranho, não para ter quebrado dessa forma. Seria bom quebrar o conteúdo do for em outro método